Documentation/sources/content-card-ui/tutorial/customizing-content-card-templates.md

```diff
@@ -0,0 +1,50 @@
+# Customizing Content Card Templates
+
+This tutorial explains how to customize the UI of content cards in your application.
+
+## Overview
+
+The Messaging extension provides a way to customize content cards using [Compose modifiers](https://developer.android.com/develop/ui/compose/modifiers?hl=en). You can customize the content card templates by defining an overriden [style](../public-classes/Styles/README.md) that is then applied to the overall template style (e.g. [SmallImageUIStyle](../public-classes/Styles/smallimageuistyle.md)).
+
+## Creating a custom style using the Compose Modifier 
```

Comment:
Same comment here. There are other fields that need to be specified apart from modifier to change the look and feel. You can check the wiki for reference
https://wiki.corp.adobe.com/display/adms/Using+Content+Cards+with+UI+on+Android#UsingContentCardswithUIonAndroid-Createcustomcontentcardstyle